wallet/src/main/res/layout/fragment_certificate_pager.xml

```diff
@@ -45,6 +45,51 @@
 			app:layout_constraintStart_toStartOf="parent"
 			app:layout_constraintTop_toTopOf="parent" />
 
+		<androidx.constraintlayout.widget.ConstraintLayout
+			android:id="@+id/certificate_page_banner"
+			android:layout_width="match_parent"
+			android:layout_height="0dp"
+			android:layout_margin="@dimen/spacing_medium_small"
+			android:background="@drawable/bg_info_banner"
+			android:clickable="true"
+			android:clipToPadding="false"
+			android:padding="@dimen/spacing_medium"
+			android:visibility="gone"
+			app:layout_constraintHeight_default="wrap"
+			app:layout_constraintTop_toTopOf="parent"
+			tools:visibility="visible">
+
+			<TextView
+				android:id="@+id/certificate_page_banner_title"
+				style="@style/CovidCertificate.Text.Bold"
+				android:layout_width="0dp"
+				android:layout_height="wrap_content"
+				android:layout_marginEnd="@dimen/spacing_medium"
+				app:layout_constraintEnd_toStartOf="@+id/certificate_page_banner_dismiss"
+				app:layout_constraintStart_toStartOf="parent"
+				app:layout_constraintTop_toTopOf="parent"
+				tools:text="@string/wallet_eol_banner_invalid_from_first_february_homescreen_title" />
```

Comment:
Can we  add android:ellipsize = "end".
The French Text, will probably be too long